CHANGELOG.md

```diff
@@ -76,8 +76,10 @@ Added the default value for the `message` parameter passed to the `MessageCompos
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added `onChannelAvatarClick` handler to `MessageListHeader`. [#4545](https://github.com/GetStream/stream-chat-android/pull/4545)
 
 ### ⚠️ Changed
+- Added `onChannelAvatarClick` handler to `MessageListHeader`. [#4545](https://github.com/GetStream/stream-chat-android/pull/4545)
```

Comment:
Does this duplication intend?

Reply:
Yes. 

It's essentially an addition, but it could also potentially be a breaking change for users who are not using named arguments.

I think it's fairly common for Compose user to use named arguments, but still wanted to warn users who don't.